README.md

```diff
@@ -1,3 +1,58 @@
-Voron Tap
+# Voron Tap
+
+![Voron-Tap3](images/Voron-Tap.gif)
+
+Tap is a nozzle-based z-probe for the V2 and Trident printer designs. The entire toolhead moves to trigger an optical switch. Tap offers many advantages:
+
+* Extreme precision 0.4μm (0.0004mm)
+* Any build surface and easily change at will
+* Durability via optical sensor (millions of probe cycles)
+* High temperature reliability (70C to 100C)
+* Simplified probe mechanics (no dock/undock macros)
+* No separate Z-endstop required
+* Crash protection
+
+## Requirements
+
+* Bed must be stable for probing force of 500-800 grams
+  * Switchwire, V1.8, Legacy, V0 beds are NOT rigid enough
+* Mounts to MGN-12H X-axis
+* Front mounted extruder (Clockwork2, LGX, Galileo)
+* Accuracy depends on good mechanical condition
+* Must have 5V available or special 24V PCB
```

Comment:
Perhaps change to 'available at toolhead or use a special 24V sensor PCB' to reduce potential ambiguity?